rust/src/lib.rs

```diff
@@ -171,8 +177,6 @@ pub mod ffi {
         fn treefile_new(filename: &str, basearch: &str, workdir: i32) -> Result<Box<Treefile>>;
 
         fn get_workdir(&self) -> i32;
-        fn get_postprocess_script_fd(&mut self) -> i32;
-        fn get_add_file_fd(&mut self, filename: &str) -> i32;
```

Comment:
Highlighting this (from the commit message):

> Note that there's one very magical line of diff here worth calling out:
We dropped an interface from the cxxrs bridge, because both sides
are now Rust! The treefile code can directly return an &mut File reference
instead of needing to pass the raw fd as i32.

Reply:
I picture cxx-rs like scaffolding around a building being renovated.  Here, one small bit of scaffolding was removed, revealing a bit of shiny new renovated building behind it.